File: backend/test/customer_test.rb
require 'test/unit'
load 'app/model/customer.rb'

class CustomerTest < Test::Unit::TestCase

  def setup
    @user = Customer.new(name="Felipe", email="felipe@teste.com", membership=nil)
  end

  def test_name
  	p @user.name
    assert_equal("Felipe", @user.name)
  end

  def test_email
  	p @user.email
    assert_equal("felipe@teste.com", @user.email)
  end

end

File: backend/proposal/core/test/customer_test.py
import unittest
from core.customer import Customer

class TestCustomer(unittest.TestCase):
    # Testing craeting customer
    def test_creation(self):
        customer = Customer("test@test.com")
        self.assertEqual(customer.email, "test@test.com")

if __name__ == "__main__":
    unittest.main()
